blueocean-dashboard/src/main/java/io/jenkins/blueocean/BlueOceanWebURLBuilder.java

```diff
@@ -124,7 +124,7 @@ private BlueOceanWebURLBuilder() {
         } else if (classicModelObject instanceof Item) {
             Resource blueResource = BluePipelineFactory.resolve((Item) classicModelObject);
             if (blueResource instanceof BlueMultiBranchPipeline) {
-                return getOrgPrefix() + "/" + encodeURIComponent(((BluePipeline) blueResource).getFullName()) + "/branches";
+                return getOrgPrefix() + "/" + encodeURIComponent(decodeURIComponent(((BluePipeline) blueResource).getFullName())) + "/branches";
```

Comment:
All this messing about with URLs is surely going to come back and bite us on the ass.